test/runtests.jl

```diff
@@ -135,3 +135,23 @@ end;
     @test sum(LightGraphs.adjacency_matrix(LightGraphs.SimpleGraph(t))) == 20
 
 end;
+
+
+@testset "SimpleHypergraphs Modularity" begin
+    Random.seed!(1234)
+    hg = Hypergraph{Bool}(10, 12)
+    for i in eachindex(hg)
+        if rand() < 0.2
+            hg[i] = true
+        end
+    end
+
+    @test findmodularity(hg,3,100) ==
+         (bp = [[3, 6], [1, 2, 4, 5, 8, 10], [7, 9]], bm = 0.2340529955954218)
```

Comment:
Are these values calculated by hand?

Reply:
Do we have the tests against more known values?

Reply:
no

Reply:
So how can you be sure that you can safely merge this PR?